consensus/src/common.rs

```diff
@@ -79,3 +79,14 @@ impl Chain<ProtocolState> for ProtocolStateChain {
         self.top().map(|s| s.hash())
     }
 }
+
+pub fn gen_spot(block: &mut ProtocolState) {
```

Comment:
If this function just does one line, better to inline it where it's called and maybe add a comment hinting that this is part of gen_spot

Reply:
Fixed.